packages/python/BUILD

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2015 Nuxi, https://nuxi.nl/
+#
+# This file is distributed under a 2-clause BSD license.
+# See the LICENSE file for details.
+
+import os
+import sysconfig
+
+def build(ctx):
+    srcdir = ctx.extract('Python-3.6.0a2')
+
+    # Delete the in-tree copy of these, to make certain the out-of-tree
+    # versions are used, i.e. those generated from our patched importlib
+    os.unlink(os.path.join(str(srcdir), 'Python', 'importlib.h'))
```

Comment:
You should be able to write this as just:

    srcdir.path('Python/importlib.h').remove()
    srcdir.path('Python/importlib_external.h').remove()